SwiftStaticLibrary/Public/SwiftStaticLibrary.h

```diff
@@ -0,0 +1,12 @@
+//
+//  SwiftStaticLibrary.h
```

Comment:
Umbrella header for the static library.  Is this somewhat equivalent to a module header for a dynamic framework?